src/main/java/com/epam/model/service/TextFileService.java

```diff
@@ -0,0 +1,110 @@
+package com.epam.model.service;
+
+import com.epam.model.entity.TextInfoHolder;
+
+import java.io.*;
+import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.epam.model.entity.TextInfoHolder.Builder;
+
+public class TextFileService {
+    private final static String TEXT_FILE = "src/data.txt";
+    private final static String SENTENCE_COUNT_REGEX = "[.]\\s";
```

Comment:
`private final static > private static final`
Please refer to https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.3.1